runtime/shared_common/j9shr.tdf

```diff
@@ -2966,3 +2964,11 @@ TraceEvent=Trc_SHR_RRM_storeNew_existingEntryRemoved Overhead=1 Level=6 Template
 TraceExit-Exception=Trc_SHR_CM_updateROMClassResource_Exit8 Overhead=1 Level=1 Template="CM updateROMClassResource: Failed to allocate memory for ROMClass resource"
 
 TraceEvent=Trc_SHR_CM_updateROMSegmentList_NewHeapAlloc Overhead=1 Level=4 Template="CM updateROMSegmentList: Updated class segment list - currentSegment is %p, new heapAlloc=%p"
+
+TraceException=Trc_SHR_CM_storeCacheUniqueID_Failed_To_Get_Manager Overhead=1 Level=1 Template="CM::storeCacheUniqueID(): Failed to get the corresponding manger"
+TraceException=Trc_SHR_CM_storeCacheUniqueID_Failed_To_Store_Prereq_UniqueID Overhead=1 Level=1 Template="CM::storeCacheUniqueID(): Failed to store the prerequisite cache unique ID %.*s"
+TraceEvent=Trc_SHR_CM_storeCacheUniqueID_generateCacheUniqueID_before Overhead=1 Level=7 Template="CM::storeCacheUniqueID(): generateCacheUniqueID() - (createTime: %llx, metadataBytes: %llx, classesBytes: %llx, lineNumTabBytes: %llx, varTabBytes: %llx) "
```

Comment:
`%llx` is not the correct type for UDATA (which is variably sized), the correct type is `%zx`. `%llx` is correct for U_64. Same problem in Trc_SHR_CC_startup_getCacheUniqueID_before